examples/usecases/incremental-training-with-layer-freezing.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>Let's add a statement that the data is random and therefore the evaluation metrics will not "make sense"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>